### ISSUE TITLE
What do the swag packages contain?

### ISSUE THREAD

**Issue by fr3fou** [education/graduation#3317]

**Comment by mitulsavani** [comment]
Let it be a surprise!

**Comment by jasperchua99** [comment]
would like to know as well will there be tracking number given for the swags?

**Comment by mitulsavani** [comment]
Hi @jashgopani, there are over [2200](https://github.com/education/graduation/pulse#merged-pull-requests) contributors whose Pull Request got merged till date!
What I mean by that is, almost all of them qualify for some swags and by looking at the number, I doubt the staff can send tracking numbers to all of us.

**Comment by jashgopani** [comment]
@mitulsavani what can I do about it  😂...why did you mention me here ?

**Comment by julianschiavo** [comment]
I'd understand if they can't, but I don't think the number of requests is really a limitation - I doubt they'll be creating shipping tags and everything manually and they've gotta keep track of the shipments themselves anyways, an automated email with a tracking number would suffice

**Comment by faizan8684** [comment]
Swags can be digital too, as given by Microsoft in Build 2020. You can check it out [here](https://github.com/microsoft/Build2020_DigitalSwag).

**Comment by jasperchua99** [comment]
agreed but i think some of us will agree that having a physical swag could show the hard work that u have put into for these few years and i think employers would be proud of our hard work rather than just a digital certificate that i just accidentally deleted a few weeks ago and cannot retrieve back

**Comment by chandresh189** [comment]
I guess there will be laptop stickers or there a T-shirt from Github

**Comment by manavendrasen** [comment]
@chandresh189 since they didn't ask us to fill a t-shirt sizing form yet. It is unlikely that there's a T-shirt. I reckon its stickers + a certificate/letter.

**Comment by pranavavva** [comment]
To be frank, I would love any sort of swag from Github. Free stuff is always awesome!

**Comment by NoobSolver** [comment]
Anyone got the swags??

**Comment by ayushbathrey** [comment]
> Anyone got the swags??

Did anyone got it?

**Comment by pranavavva** [comment]
Nope.

**Comment by chandresh189** [comment]
Not yet

**Comment by amansharma07** [comment]
[#4456 (comment)](https://github.com/education/graduation/issues/4456#issuecomment-663642487)
Hope it helps! :)

**Comment by AlexanderOMara** [comment]
For Tier 1 I got a variety of sticks (https://github.com/education/graduation/issues/4456#issuecomment-663894825).

**Comment by fr3fou** [comment]
what kind of sticks?

**Comment by AlexanderOMara** [comment]
> what kind of sticks?

(4 DEV stickers (one shaped like a graduation cap) and 1 sloth (which I'm not sure what it is supposed to represent)](https://i.imgur.com/JwE6ACW.jpg).

**Comment by pranavavva** [comment]
Just got my swag!! They are shipping out of St. Louis, MN if anyone’s curious. 

Respectfully,
Pranav Avva
